### ISSUE TITLE
TOB info needs to be updated

### ISSUE THREAD

**Issue by dret** [OAI/OpenAPI-Specification#3482]
The list at https://github.com/OAI/OpenAPI-Specification/blob/main/TOB.md is outdated and should be updated.

**Comment by handrews** [comment]
in the last TDC meeting of 2023, we agreed that the TOB in general needs updating.  Particularly regarding coordinating across multiple specs (OAS, Workflows, Overlays).

**Comment by darrelmiller** [comment]
I agree that the outdated information about the TOB needs to be fixed.  I'm not yet convinced that the TOB in its current form should be the group responsible for coordinating across sigs.  The process for managing the membership of the TOB is overly complex.

**Comment by lornajane** [comment]
See also #3356 . Seems like we know we need to do something else, but it isn't clear what the next steps would be.

**Comment by baywet** [comment]
@hellobudha will bring this up to the governance meeting next week and share some updated after, Thanks! :)

**Comment by baywet** [comment]
update: the BGB meeting got postponed, most likely to happen next week

**Comment by hellobudha** [comment]
We initiated this discussion at the Business Governance Board meeting yesterday (Jan 28, 2026) along with @earth2marsh 
Looking for your inputs regarding the following discussion points:

1. Are the roles and responsibilities across the TSC and TOB clear?
2. How do we ensure cohesion across the 3 specs?
3. How do we simplify decision-making? How can we make the process more efficient?
4. Should the TOB still serve as an escalation/ de-escalation point?
5. Should each technical stream have its own chair/co-chair? With TOB or a Technical Governance Board ensuring governance and acting as an escalation point if needed?

Once we reach (a degree of) consensus across the TSC and BGB regarding the best way to support and manage the different technical streams and governance, we will ratify it as a change in the OpenAPI Initiative's Charter.

**Comment by handrews** [comment]
Moonwalk 2026 discussion now has a list of links to AI/LLM-related glossaries so we can learn the terminology, and figure out which concepts/terms we need to integrate into the various OpenAPI Initiative specifications.

**Comment by lornajane** [comment]
The TOB is a bit unclear but also is not in use so I don't see that as a huge problem. Chairs/co-chairs is something I know is common and useful in other projects and I'd like to hear more about what that entails. Right now I personally feel that TSC has a good setup where we essentially tag team with whichever volunteer has the time/energy/bandwidth for each available thing that comes up. Since we're all pretty limited on bandwidth, everyone does a bit and it works out. I'm not sure how a chair would change that dynamic and would like to hear more about that idea. Is the idea that all the SIGs also have a chair, and we have formal co-ordination between them rather than the projects reporting into the weekly TDC meeting when they want to touch base?
